packages/apps/src/app.process.ts

```diff
@@ -154,11 +154,12 @@ export async function $process(this: App, sender: ISender, event: IActivityEvent
   try {
     let res = await next();
 
+    await context.stream.close();
+
     if (!res || !isInvokeResponse(res)) {
       res = { status: 200, body: res };
     }
```

Comment:
NIT: does this matter?

Reply:
just thought it would be easier to read if we close the stream, ie finish sending everything and hit any errors that may occur, before doing response stuff. Won't affect code logically.